Cargo.toml

```diff
@@ -33,7 +33,7 @@ wasm_bindgen = ["js-sys", "wasm-bindgen", "interface"]
 interface = []
 
 [dependencies]
-abe_policy = { path = "../abe_policy", features = ["wasm_bindgen"]}
+abe_policy = { git = "https://github.com/Cosmian/abe_policy.git", branch = "fix/interfaces", features = ["wasm_bindgen"]}
```

Comment:
just a comment to not forget to update it

Reply:
Changed.